docs/LV_client/shared_libraries/mono_lane_follower_lights.md

```diff
@@ -0,0 +1,34 @@
+# mono_lane_follower_lights.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_lane_follower_lights.png"/>
+</p>
+
+### Description
+
+This example shows how to use the monoDrive client to use a a compiled DLL to create a lane follower controller. This example allows the user to configure and update the lights attached to the ego vehicle. 
+
+For technical support contact us at <b>support@monodrive.io</b> 
+
+### Inputs
+
+- **Select a map:**  Select the map to load on the Simulator
+ 
+
+- **error in (Error Cluster):** Accepts error information wired from previously called VIs. This information can be used to decide if any functionality should be bypassed in the event of errors from other VIs. 
+
+### Outputs
+
+- **Status:**  error in can accept error information wired from VIs
+previously called. Use this information to decide if any
+functionality should be bypassed in the event of errors from
+other VIs.
+
+Right-click the error in control on the front panel and
+select Explain Error or Explain Warning from the shortcut
+menu for more information about the error.
+ 
```

Comment:
1. This could read a little bit easier with quotes around the titles of UI elements. For instance after status: "error in" can accept... and "Explain Error". 
2. The indention of the second paragraph of this bullet point is not lined up correctly. An easy fix would be to remove the space and making it part of the first paragraph. This has given me some problems in the past, let me know how I can help. 
![Screen Shot 2020-11-18 at 10 28 09 AM](https://user-images.githubusercontent.com/41026694/99557872-c3ab3300-2988-11eb-8d1b-bcd0c803a177.png)

Reply:
Good catch. This is a bug in my documentation generator.